osu.Server.Queues.ScoreStatisticsProcessor/Commands/ScoreCommands.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Threading;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using McMaster.Extensions.CommandLineUtils;
+using osu.Server.Queues.ScoreStatisticsProcessor.Commands.Score;
+
+namespace osu.Server.Queues.ScoreStatisticsProcessor.Commands
+{
+    [Command(Name = "score", Description = "Runs batch processing on score totals for scores and users.")]
+    [Subcommand(typeof(ChangeModMultiplierCommand))]
+    public class ScoreCommands

```

Comment:
I would have probably just put the command in maintenance, given there's already similar score repair commands in there?

Reply:
At the time of writing I wasn't sure if there were going to be any others, for e.g. user total recalculations. But I think it's pretty clear to me that it's not required anymore, will move.

Reply:
https://github.com/ppy/osu-queue-score-statistics/pull/246/commits/2125c1887481dbf581005d366624e2f0d88f65cb